packages/trader/src/Modules/MT5/Components/mt5-account-card.jsx

```diff
@@ -1,30 +1,32 @@
 import classNames from 'classnames';
 import React from 'react';
-import { Money, Button } from '@deriv/components';
+import { Money, Button, Card } from '@deriv/components';
 import { Localize } from '@deriv/translations';
 import { Mt5AccountCopy } from './mt5-account-copy.jsx';
-import { getMT5WebTerminalLink } from '../Helpers/constants';
+// import { getMT5WebTerminalLink } from '../Helpers/constants';
 
 const LoginBadge = ({ display_login }) => (
-    <div className='mt5-account-card__login'>
+    <span className='mt5-account-card__login'>
```

Comment:
Could we leverage @sara-fs new `Text` component here?

Reply:
Yes and the other `<p>`s as well 🙏